### ISSUE TITLE
Support multiple entry files/patterns/directories

### ISSUE THREAD

**Issue by zinserjan** [zinserjan/mocha-webpack#45]
At the moment mocha-webpack supports only a single argument (file, directory or pattern).

We should support multiple entries to support the following cases:
- code is organized into multiple src folders (e.g. src & lib) #27 
- code coverage reports for whole codebase, not just with files that are referenced in test files #19 
- allow execution of individual test files #8

Examples:

``` bash
# run all test files in src & lib folder
mocha-webpack src/**/__test__/*.spec.js lib/**/__test__/*.spec.js

# run only a few tests
mocha-webpack test/es6/Browser.spec.js test/es6/ErrorHandler.spec.js src/templates/TemplateComponent.spec.js
```

**Comment by fairbairn** [comment]
+1

- what would also be a nice feature is the ability to specify a file containing the specific test files to include in the bundle for execution.

mocha-webpack --bundle myfiles.txt

myfiles.txt
--------------
test/test1-spec.js
test/test3-spec.js
test/nested/someothertest.js

**Comment by arthurio** [comment]
@zinserjan This feature is exactly what we need to leverage circle ci test auto-balancing, do you have any estimate on when this would be available? If nothing has been planned yet, can you give me some guidance to make it happen? So far, from looking at the source code, it looks like the bulk of the changes need to happen in the `prepareWebpack.js` file and set the entry as a list of the files passed as arguments instead of using one file and setting the context with its path + the entry with its basename. Then I think what's left would be to update `parseArgv.js` and remove `.demand(0, 1)` on line #212.

**Comment by pkuczynski** [comment]
This is at least in the scope of @zinserjan suggestion fixed in #79. Tested with `"mocha-webpack": "1.0.0-beta.1"`
